src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -798,17 +794,16 @@ public void scrollToPosition(String selector) {
     jsActions.scrollToSpecificElement(driver.findElement(By.cssSelector(selector)));
     PageObjectLogging.log("scrollToSelector", "Scroll to the web selector " + selector, true);
   }
-  // This scroll has been created because ad is not displayed if we scroll quickly to the Footer ADEN-4359
-  public void scrollToBottomLeaderboard(){
-    scrollToFooter();
-    wait.forElementVisible(By.cssSelector(".editarea"));
-    scrollToFooter();
-  }
 
-  public void clickOnAdImage(String slotName){
-    adSelector(slotName);
-    adSelector(slotName).click();
-    PageObjectLogging.log("clickOnAdImage", slotName + " is clicked", true);
+  // This scroll has been created because ad is not displayed if we scroll quickly to the Footer ADEN-4359
+  public void scrollToSlot(String slotName) {
```

Comment:
I don't think we need this method. Are there any reasons we can't wait for the slots being filled?